tests/release/e2e-test-default-bundle.go

```diff
@@ -62,9 +58,6 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-738]test-release-service-default-
 		_, err = fw.AsKubeAdmin.ReleaseController.CreateReleasePipelineRoleBindingForServiceAccount(managedNamespace, managedServiceAccount)
 		Expect(err).NotTo(HaveOccurred())
 
-		_, err = fw.AsKubeAdmin.CommonController.CreateRegistryAuthSecret(hacbsReleaseTestsTokenSecret, devNamespace, sourceAuthJson)
```

Comment:
why is this being removed? It doesn't seem related to timeouts at all

Reply:
in the e2e-tests framework there was a PR that creates this secret in userframework , 
so it's created by deployment and if I keep it it will fail with exception of AlreadyExist

Reply:
Then can you mention that in the commit message or make it in its own commit or something?